Tests/Common/Securities/CashTests.cs

```diff
@@ -677,8 +677,8 @@ private static TimeKeeper TimeKeeper
 
             // USDT (Tether) Cases
             new object[] { new BitfinexBrokerageModel(), Currencies.USD, "USDT", false, new[] { Symbol.Create("USDTUSD", SecurityType.Crypto, Market.Bitfinex) } },
-            new object[] { new BitfinexBrokerageModel(), Currencies.EUR, "USDT", false, new[] { Symbol.Create("BTCUSDT", SecurityType.Crypto, Market.Bitfinex), Symbol.Create("BTCEUR", SecurityType.Crypto, Market.Bitfinex) } }, // No USDTEUR, but indirect conversion exists
-            new object[] { new BitfinexBrokerageModel(), Currencies.GBP, "USDT", false, new[] { Symbol.Create("BTCUSDT", SecurityType.Crypto, Market.Bitfinex), Symbol.Create("BTCGBP", SecurityType.Crypto, Market.Bitfinex) } }, // No USDTGBP, but indirect conversion exists
+            new object[] { new BitfinexBrokerageModel(), Currencies.EUR, "USDT", false, new[] { Symbol.Create("EURUSDT", SecurityType.Crypto, Market.Bitfinex) } }, 
+            new object[] { new BitfinexBrokerageModel(), Currencies.GBP, "USDT", false, new[] { Symbol.Create("GBPUSDT", SecurityType.Crypto, Market.Bitfinex) } }, 
```

Comment:
yeah, but for the cases when direct conversions aren't available. `Cash.EnsureCurrencyDataFeed` fetches the direct conversion symbols for instance `EURUSDT` is now available in SPDB.csv. So, it will not use the indirect conversion pair `BTCEUR` & `BTCUSDT`